.swiftlint.yml

```diff
@@ -0,0 +1,23 @@
+disabled_rules:
+  - cyclomatic_complexity
+  - identifier_name
+  - multiple_closures_with_trailing_closure
+  - nesting
+  - type_name
+  - unused_closure_parameter
+
+excluded: # case-sensitive paths to ignore during linting. Takes precedence over `included`
+  - Pods
+  - "**/*.pb.swift" # exclude files with a wildcard
+
+line_length: 300
```

Comment:
I doubt we have lines longer than 120 characters, but... if it was not enforced or checked, who knows.